src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+     action="/" 
+     method="GET"
+     data-qa="big"
+     class="form form--big"
+     > 
+      <label >
+        <img 
+          src="images/Search.svg" 
+          class="iconBig"
+          alt="search image"
+          >
+        <input 
```

Comment:
between close & open tags we put an empty line
use it in all cases
```suggestion
          >
          
        <input 
```
this is more readable

Reply:
please fix the closing tag